yarn.lock

```diff
@@ -2175,6 +2175,24 @@
   dependencies:
     slash "^3.0.0"
 
+"@rollup/plugin-strip@^1.3.2":
```

Comment:
Are the changes to `yarn.lock` necessary?

Reply:
they're specific to our branch, i reverted the yarn.lock when I rebased against master yesterday